extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,269 @@
+  align-items: center !important;
+  z-index: 1 !important;
+  transition: background 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container {
+  background: black !important;
+  width: min(650px, 90%) !important;
+  border-radius: 10px !important;
+  padding: 40px !important;
+  position: relative !important;
+  transition: transform 0.2s var(--i-amphtml-story-dev-tools-material-easing), opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+  transform: translateY(5vh) !important;
+  opacity: 0 !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] {
+  background: var(--i-amphtml-story-dev-tools-device-gray-transparent-1) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-bg[active] .i-amphtml-story-dev-tools-device-dialog-container  {
+  transform: translateY(0) !important;
+  opacity: 1 !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1 {
+  font-size: 12px !important;
+  text-transform: uppercase !important;
+  opacity: .4 !important;
+  letter-spacing: 1px !important;
+  font-weight: normal !important;
+  margin: 0px !important;
+  margin-bottom: 5px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container h1:not(:first-child) {
+  margin-top: 25px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container.i-amphtml-story-dev-tools-device-dialog-add-devices h1 {
+  margin-inline-start: 15px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link {
+  color: var(--i-amphtml-story-dev-tools-device-white-transparent-3) !important;
+  text-decoration: none !important;
+  display: block !important;
+  padding: 10px 0 !important;
+  transition: color 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: space-between !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link svg {
+  vertical-align: middle !important;
+  opacity: 0.6 !important;
+  transition: opacity 0.2s var(--i-amphtml-story-dev-tools-material-easing), transform 0.2s var(--i-amphtml-story-dev-tools-material-easing) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:hover {
+  color: white !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:hover svg {
+  opacity: 1 !important;
+  transform: translateX(4px) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-link:not(:last-child) {
+  border-bottom: 1px solid var(--i-amphtml-story-dev-tools-device-white-transparent-1) !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-container .i-amphtml-story-dev-tools-device-chip {
+  margin: 4px !important;
+}
+
+.i-amphtml-story-dev-tools-device-dialog-close {
+  cursor: pointer !important;
+  padding: 10px !important;
+  position: absolute !important;
+  top: 0 !important;
+  right: 0 !important;
+  opacity: .6 !important;
+  box-sizing: content-box !important;
```

Comment:
Re: all or nothing use content-box. I thought we changed it so everything uses content-box?

Reply:
Left this from a while ago, but doesn't actually do anything so will remove this line